unity-environment/Assets/ML-Agents/Scripts/CoreBrainInternal.cs

```diff
@@ -158,6 +160,11 @@ public void InitializeCoreBrain(MLAgents.Batcher brainBatcher)
                 {
                     hasPrevAction = true;
                 }
+
+                if (graph[graphScope + "masked_actions"] != null)
```

Comment:
- "masked_scope" should be a static variable.
- replace with: `hasMaskedActions = graph[graphScope + "masked_actions"] != null`

Reply:
Can you explain the first bullet? A static variable would pose problems if it is different for multiple Internal Brains. Am I missing something?